### PR TITLE
Restore the header row

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: src/listing.ts [hunk 1]
```diff
@@ -156,9 +177,23 @@ class DirListing extends Widget {
...
+    let contents = document.createElement('ul');
+    header.className = LIST_HEADER_CLASS;
+    body.className = LIST_CONTAINER_CLASS;
+    contents.className = LIST_AREA_CLASS;
+    node.appendChild(header);
+    node.appendChild(body);
+    body.appendChild(contents);
+    let files = document.createElement('span');
+    let modified = document.createElement('span');
+    files.textContent = 'Files';
```
Comment (afshin) [inline src/listing.ts:191]: Is this language going to be passed in / configurable at some point?

File: src/listing.ts [hunk 2]
```diff
@@ -156,9 +177,23 @@ class DirListing extends Widget {
...
+    files.className = HEADER_FILE_CLASS;
+    modified.textContent = 'Last Modified';
```
Comment (afshin) [inline src/listing.ts:193]: Same question: is this language going to be passed in / configurable at some point?

**Comment by afshin** [comment]
Looks good to me, I just had those two comments. I'm happy to merge if we are keeping the language inside the class for now.

**Comment by blink1073** [comment]
We could have a static mapping of textcontent and titles on the main browser class that could be swapped out, which is in turn passed to the sub-classes.

**Comment by afshin** [comment]
That's a good strategy, yeah. Since _most_ of our widgets don't have language in them, this isn't coming up too frequently, but something like what you're suggesting sounds good to me.

**Pull request merged**